### PR TITLE
Added folder for sample data and included instructions for the Australian PBS medicines 

### PR DESCRIPTION
**Pull request by fgiorlando**
Added a folder for sample-data.

- created folder
- created Readme with description of how to use PBS database

*Note*: I had hoped to include the processed .csv file but realised that it is copyrighted.  Therefore, I included links and instructions on how to use the PBS database file.

### CODE REVIEW

File: sample_data/Readme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This folder contains sample data sets which import with the import command fromt the server repository, e.g.: `utils/inv-import.js file.csv YYYY-MM-DD`
+
+### Medications
+
+source | filename | descritpion | header information | notes 
+-------|----------|-------------|--------------------|-------
+http://www.pbs.gov.au/browse/downloads | amt_20170201.csv | Australian PBS database | see [here](https://dev.pbs.gov.au/readme.html) | this database includes all Australian PBS drugs.  The delimiter is `!`. To use the file it needs to be slightly modified (commas replaced with semi-colons).
```
Comment (jkleinsc) [inline sample_data/Readme.md]: Looking briefly at the data, it appears that the column header names in the CSV files need to modified as well in order for the import tool to understand the data being imported in.

File: sample_data/Readme.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+This folder contains sample data sets which import with the import command fromt the server repository, e.g.: `utils/inv-import.js file.csv YYYY-MM-DD`
```
Comment (jkleinsc) [inline sample_data/Readme.md]: Minor spelling issue here.  Should be _from_ not _fromt_

File: sample_data/Readme.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+### Medications
+
+source | filename | descritpion | header information | notes
```
Comment (jkleinsc) [inline sample_data/Readme.md]: Another minor spelling issue, should be _description_, not _descritpion_

**Comment by fgiorlando** [comment]
I have updated the pull request and moved it to the hospitalrun-server repo.
- [x] fixed issue with naming of folder (now `sample-data`)
- [x] added image to `Readme.md` to better describe column layout of PBS database

**Comment by jkleinsc** [comment]
Closing this PR in favor of HospitalRun/hospitalrun-server#42

**Pull request closed without merging**